client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx

```diff
@@ -0,0 +1,163 @@
+import { useLocation } from 'react-router';
+import type {
+	CancellationContextInterface,
+	CancellationPageTitleInterface,
+	CancellationRouterState,
+} from '@/client/components/mma/cancel/CancellationContainer';
+import {
+	CancellationContext,
+	CancellationPageTitleContext,
+} from '@/client/components/mma/cancel/CancellationContainer';
+import { linkCss } from '@/client/components/mma/upgrade/UpgradeSupportStyles';
+import {
+	buttonCentredCss,
+	stackedButtonLayoutCss,
+} from '@/client/styles/ButtonStyles';
+import { DATE_FNS_LONG_OUTPUT_FORMAT, parseDate } from '@/shared/dates';
+import type { PaidSubscriptionPlan } from '@/shared/productResponse';
+import { getMainPlan } from '@/shared/productResponse';
+import { calculateBillingPeriod } from '@/shared/productTypes';
+import {
+	headingCss,
+	iconListCss,
+	listWithDividersCss,
+	whatHappensNextIconCss,
+} from '../../../../../styles/GenericStyles';
+
+export const DigiSubDiscountConfirm = () => {
+	const pageTitleContext = useContext(
+		CancellationPageTitleContext,
+	) as CancellationPageTitleInterface;
+
+	const cancellationContext = useContext(
+		CancellationContext,
+	) as CancellationContextInterface;
+
+	const location = useLocation();
+	const routerState = location.state as CancellationRouterState;
+	const digiSub = cancellationContext.productDetail;
+
+	const mainPlan = getMainPlan(digiSub.subscription) as PaidSubscriptionPlan;
+
+	const priceDisplay = `${mainPlan.currency}${mainPlan.price / 100}`;
+
+	const nextBillingDate = parseDate(
+		mainPlan.chargedThrough ?? undefined,
+	).dateStr(DATE_FNS_LONG_OUTPUT_FORMAT);
+
+	const userEmailAddress = routerState?.user?.email;
+
+	const billingPeriod = calculateBillingPeriod(
+		mainPlan.billingPeriod,
+	).toLowerCase();
+
+	useEffect(() => {
+		pageTitleContext.setPageTitle('Your subscription');
+	}, []);
+
+	return (
+		<>
+			<section
+				css={css`
+					padding-top: ${space[4]}px;
+					margin-bottom: 32px;
+				`}
+			>
+				<h1 css={headingCss}>Discount confirmed</h1>
+				Thank you for continuing to fund our journalism.
+			</section>
+
+			<section
+				css={css`
+					margin-top: 20px;
+					border-bottom: 1px solid ${palette.neutral[86]};
+					border-top: 1px solid ${palette.neutral[86]};
+					padding-bottom: ${space[5]}px;
+				`}
+			>
+				<Stack space={5}>
+					<ul
+						css={[
+							iconListCss,
+							listWithDividersCss,
+							whatHappensNextIconCss,
+						]}
+					>
+						<li>
+							<SvgEnvelope size="medium" />
+							<span>
+								<strong
+									css={css`
+										padding-bottom: ${space[1]}px;
+									`}
+								>
+									Check your email
+								</strong>
+								<br />
+								We have sent you a discount confirmation to{' '}
+								{userEmailAddress}
+							</span>
+						</li>
+						<li>
+							<SvgGift size="medium" />
+							<span>
+								<strong
+									css={css`
+										padding-bottom: ${space[1]}px;
+									`}
+								>
+									25% discount for 3 months
```

Comment:
is it always 3 months 🤔

Reply:
I thought so...This screen still needs data inputted so I'm going to leave it and we can ask Daniel when it comes to inputting the other values

Reply:
Daniel is going to confirm so I have popped in a placeholder